examples/dispatch-custom-event/README.md

```diff
@@ -0,0 +1,31 @@
+# Fire a Custom Event
+
+**Test Source**: [index.js](index.js)
+
+**Page Source**: [index.html](index.html)
+
+This example shows how to fire DOM [events](https://developer.mozilla.org/en-US/docs/Web/API/Event) with TestCafe.
+
+We use the `t.dispatchEvent` method to fire a `touchstart` event.
+
+The page displays the following information when we trigger a [touchstart event](https://developer.mozilla.org/en-US/docs/Web/Events/touchstart):
+
+* the event's constructor
+* whether the event is [cancelable](https://developer.mozilla.org/en-US/docs/Web/API/Event/cancelable)
+* whether the event [bubbles](https://developer.mozilla.org/en-US/docs/Web/API/Event/bubbles)  
+
+TestCafe invokes the [CustomEvent Constructor](https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/CustomEvent) to create events, unless the event in question is a [Mouse event](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/dispatchevent.html#mouse-events), a [Keyboard event](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/dispatchevent.html#keyboard-events), an [Input event](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/dispatchevent.html#input-events), a [Focus event](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/dispatchevent.html#focus-events) or a [Pointer event](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/dispatchevent.html#pointer-events).
+
+Include the `eventConstructor` option to use the [TouchEvent Constructor](https://developer.mozilla.org/en-US/docs/Web/API/TouchEvent/TouchEvent) instead.```
```

Comment:
Can you clarify what you mean by "Include"? Are you enabling this option ("Enable the...")? Or do you mean "Add the `eventConstructor` option"?

Reply:
You pass the option to a method, like so:

```
t.dispatchEvent(..., '...', {eventConstructor: 'EventName})
```

Reply:
Does this work

```suggestion
To use the [TouchEvent Constructor](https://developer.mozilla.org/en-US/docs/Web/API/TouchEvent/TouchEvent), Pass the `eventConstructor` option to `t.dispatchEvent`.```
```

Reply:
Thank you. Please apply your suggestion.